### PR TITLE
Start Transifex sync script

### PR DESCRIPTION
**Pull request by samuellefever**
I started the script using github3 library.
It gets the translations, compares it with what is in the repository, and do the commit only if needed.

I am blocked with a error on line 88 when I try to commit.
```
github3.exceptions.UnprocessableEntity: 422 Invalid request.
"email", "name" weren't supplied.
```

Any help is welcome.

### CODE REVIEW

**Comment by max3903** [comment]
You need to provide an identity to commit. Use "Transbot <transbot@odoo-community.org>".

File: tools/tx_pull.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+"""
+This script is meant to pull the translations from Transiflex .
+Technically, it will pull the translations from transiflex,
```
Comment (max3903) [inline tools/tx_pull.py]: Transifex, no 'l'

**Comment by yvaucher** [comment]
@samuellefever Isn't it your git author configuration that is missing an email? 
git config --global user.email "email"

File: tools/tx_pull.py (hunk 1 above)
Comment (samuellefever) [inline tools/tx_pull.py]: I tries with author = {'name' : 'samuellefever', 'email' : 'sam@niboo.be'} and it stays the same

**Comment by samuellefever** [comment]
As I am not using my local git config but the github3 library, it did not fix the problem

**Comment by max3903** [comment]
If I am not mistaken, @pedrobaeza told me today that he will test this script.

**Comment by bwrsandman** [comment]
@pedrobaeza how goes the test?

File: tools/tx_pull.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+"""
+This script is meant to pull the translations from Transifex .
+Technically, it will pull the translations from transifex,
+compare it with the po files in the repository and replace it if needed
+"""
```
Comment (max3903) [inline tools/tx_pull.py]: Could you please document here how to install its dependencies (Transifex), how to configure it and how use it ?

**Comment by pedrobaeza** [comment]
Superseeded by #69

**Pull request closed without merging**